packages/core-js/internals/object-get-own-property-symbols-external.js

```diff
@@ -0,0 +1,6 @@
+var toIndexedObject = require('../internals/to-indexed-object');
+var nativeGetOwnPropertySymbols = require('../internals/object-get-own-property-symbols').f;
+
+module.exports.f = function getOwnPropertySymbols(it) {
+  return nativeGetOwnPropertySymbols(toIndexedObject(it));
+};
```

Comment:
I think that it can be moved to `es.symbol` since it's used only here.

Reply:
`toIndexedObject` required only for old IE where we can access to string chars by index, those engines haven't native `Object.getOwnPropertySymbols`, so here should be used `toObject`.

Reply:
Understood. I just got the test suite working. I'll commit some updates in the next day or two.

Reply:
Moved `packages/core-js/internals/object-get-own-property-symbols-external.js` into `es.symbol` and switched `toIndexedObject` to `toObject`. Also fixed failing core-js-pure unit tests while addressing `packages/core-js/internals/object-get-own-property-symbols-external.js`